Sources/NFCPassportReader/NFCPassportModel.swift

```diff
@@ -70,6 +69,14 @@ public class NFCPassportModel {
         return telephone
     }()
 
+    public private(set) lazy var typeOfDocument: DocumentType = {
+        return DocumentType.toDocumentType(code: documentType)
+    }()
+
+    public var isNorwegianEmergencyPassport: Bool {
+        return typeOfDocument == .norwegianEmergencyPassport
+    }
```

Comment:
It will return true if `setAsNorwegianEmergencyPassport(documentNumber: String)` has been called, which is the only function that explicitly sets `typeOfDocument`. I'm not super happy with how it's implemented as it is confusing, do you have any other solutions?

Reply:
Aah never mind, I miss read this and thought this was a computed property!